### ISSUE TITLE
Error while loading html dashboard

### ISSUE THREAD

**Issue by EkaX** [lucidworks/banana#366]
Hi when I accessed **http://localhost:8983/solr/banana/src/index.html** that only came out on a white screen or had a solution to install banana dashboard on Solr 8.5?

Thanks

**Comment by aadel** [comment]
This [PR](https://github.com/lucidworks/banana/pull/367) addresses CSP script requirements introduced in Solr 8.4 and fixes this issue.

**Comment by EkaX** [comment]
Hi Thanks aadel my problem is solved but the problem is still same
i try bypass the CSP with a plugin in google chrome https://chrome.google.com/webstore/detail/disable-content-security/ieelmcmcagommplceebfedjlakkhpden

**Comment by jason-missiontech** [comment]
We are having the same issue with Solr 8.6.2, even with those changes.

**Comment by Fractal2017** [comment]
I found that it works for IE 10
![image](https://user-images.githubusercontent.com/29259883/107907810-aa336c00-6f22-11eb-817a-1d7262541507.png)

**Comment by kenghoon** [comment]
Right. Tested on Solr 8.8.1. It works on IE 10 (Win 10) using the original banana .zip from https://github.com/lucidworks/banana.
Chrome, Mozilla and Edge display blank.

**Comment by minesh-melvin** [comment]
Had the same problem with 8.8.2
